crates/sovereign-sdk/rollup-interface/src/state_machine/zk/mod.rs

```diff
@@ -50,7 +52,8 @@ pub trait ZkvmHost: Zkvm + Clone {
     /// with some mild performance overhead and is not as easy to debug as [`simulate_with_hints`](ZkvmHost::simulate_with_hints).
     fn run(&mut self, with_proof: bool) -> Result<Proof, anyhow::Error>;
 
-    /// Extracts public input form the proof.
+    /// Extracts public input and receipt from the proof.
+    #[allow(clippy::type_complexity)]
```

Comment:
no need for this now that you reverted the changes I assume?

Reply:
nice catch, removed